File: POO/Cpp/Lista 01/l1q7.cpp
#include <iostream>

using namespace std;

int main() {
    float f;
    cout << "Escreva o valor em Fahrenheit: ";
    cin >> f;

    float c = (5*(f-32))/9;
    cout << "o valor em Celsius é " << c << endl;

    return 0;
}

File: POO/Rust/Lista01/l1q7.rs
use std::io;

fn main() {
    println!("Escreva o valor em Fahrenheit");

    let mut input = String::new();
    io::stdin().read_line(&mut input).expect("Falha ao ler a linha");
    match input.trim().parse::<f64>() {
        Ok(f) => {
            println!("Valor em C: {}", (5.0 * (f - 32.0)) / 9.0);
        }
        Err(_) => {
            println!("Valor inválido!");
        }
    }

    return

}